ntbs-service/Pages/Notifications/OverviewPartials/_ClusterInformationOverviewPartial.cshtml

```diff
@@ -0,0 +1,8 @@
+@model ntbs_service.Models.Entities.Notification
+
+@if (Model.ClusterId != null)
+{
+    <div class="notification-overview-type-and-edit-container">
+        <h3 class="notification-details-type"> Cluster ID: <a href="/Reports"> @Model.ClusterId </a> </h3>

```

Comment:
Should this pass the cluster id. I wasn't sure about the configurable link, and seems like all requests should go through Reports page

Reply:
Yeah for now I would potentially navigate to `Reports/Cluster/id` and maybe wire in the use of `id` which isn't currently implemented.
It's a bit of a stop gap, and when we know how these should be routed out - it should be easy to include this in Reports/Index.cshtml.cs (I left a comment in there to this effect).